src/paulscode/android/mupen64plusae/PlayMenuActivity.java

```diff
@@ -71,11 +71,12 @@
     private static final String CATEGORY_GAME_SETTINGS = "categoryGameSettings";
     private static final String CATEGORY_CHEATS = "categoryCheats";
     
-    private static final String ACTION_RESUME = "actionResume";
-    private static final String ACTION_RESTART = "actionRestart";
+    public static final String ACTION_RESUME = "actionResume";
```

Comment:
Had to make these public for now so game loading can still be routed through PlayMenuActivity. This activity is eventually going to be removed entirely, but for now we need some hacks to get PlayMenuActivity to do whatever GalleryActivity tells it to do.